https://bugs.gentoo.org/939142

Exact cause unknown, but running qbs in portage's sandbox hangs
for a few people while running ldconfig, and using "env" seemingly
helps. Better (potentially upstreamable) patch welcome if someone
can figure out the right fix.
--- a/share/qbs/modules/cpp/LinuxGCC.qbs
+++ b/share/qbs/modules/cpp/LinuxGCC.qbs
@@ -48,7 +48,7 @@ UnixGCC {
             var paths = [];
             var ldconfig = new Process();
             try {
-                var success = ldconfig.exec("ldconfig", ["-vNX"]);
+                var success = ldconfig.exec("env", ["ldconfig", "-vNX"]);
                 if (success === -1)
                     return;
                 var line;
